Aptoide/aptoide-client-v8
# [MOB-175] Notification Dark Theme

**What does this PR do?**

   Adds notification reminder for dark theme.

**Database changed?**

   Yes

**How should this be manually tested?**

  Check if the notification appears 2 minutes after opening the app without showing the home notification. If the home notification appears, the notification should be canceled and not shown and vice-versa. Also check if the notification buttons are working and that events are being sent.

**What are the relevant tickets?**

  [MOB-175](https://aptoide.atlassian.net/browse/MOB-175)

**Code Review Checklist**
